genomesubsampler/prototypeSelection.py

```diff
@@ -329,13 +391,7 @@ def prototype_selection_constructive_protoclass(dm, num_prototypes, steps=100):
     def prototype_selection_constructive_protoclass(dm: DistanceMatrix,
     num_prototypes: int, steps=100: int) -> List[str]:
     '''
-    if num_prototypes < 2:
-        raise ValueError(("'num_prototypes' must be >= 2, since a single "
-                          "prototype is useless."))
-    if num_prototypes >= len(dm.ids):
-        raise ValueError(("'num_prototypes' must be smaller than the number of"
-                          " elements in the distance matrix, otherwise no "
-                          "reduction is necessary."))
+    _validate_parameters(dm, num_prototypes)
```

Comment:
no seedset for this function?? Can you open an issue for that such that I can work on it later on?